test/libraries/Pool.t.sol

```diff
@@ -17,7 +17,10 @@ contract PoolTest is Test {
 
     Pool.State state;
 
-    function testPoolInitialize(uint160 sqrtPriceX96, uint16 protocolFee, uint24 dynamicFee) public {
+    uint24 constant MAX_PROTOCOL_FEE = 1000; // 0.1%
```

Comment:
would rather this was
```
uint24 constant MAX_PROTOCOL_FEE = ProtocolFeeLibrary. MAX_PROTOCOL_FEE
```

So that if ever it changes it updates here too

Same with the one below and SwapFeeLibrary.MAX_SWAP_FEE